Feature/task 60 pozdeeva

OxanaPozdeeva opened the pull request.

OxanaPozdeeva committed:
Merge branch 'master' of https://github.com/dha01/IS

OxanaPozdeeva committed:
Merge branch 'master' into feature/Task_14_Pozdeeva

OxanaPozdeeva committed:
Merge branch 'master' of https://github.com/dha01/IS into feature/Task_14_Pozdeeva

OxanaPozdeeva committed:
ПЕ-21б Поздеева О.В.: Создание класса TeamItem

Создала класс для хранения о сущности групп.

dha01 commented on IS.Model/IS.Model/Item/Team/TeamItem.cs:
```diff
@@ -0,0 +1,30 @@
+		/// <summary>
+		/// Дата создания группы.
+		/// </summary>
+		public DateTime? Create_date { get; set; }
```
CreateDate

dha01 commented on IS.Model/IS.Model/Item/Team/TeamItem.cs:
```diff
@@ -0,0 +1,30 @@
+		/// <summary>
+		/// Идентификатор образовательной программы.
+		/// </summary>
+		public int Specialty_detail { get; set; }
```
"кроме полей с вторичными ключами"
Это именно такое поле, у него есть (должен быть) вторичный ключ в БД. Оно здесь не требуется. Специальность для группы будет получаться другим путем.

dha01 commented on IS.Model/IS.Model/Item/Team/TeamItem.cs:
```diff
@@ -0,0 +1,30 @@
+	/// Класс для хранения данных о сущности групп.
+	/// </summary>
+	public class TeamItem
+    {
```
Пробелы. Должна быть табуляция.

OxanaPozdeeva committed:
ПЕ-21б Поздеева О.В.: Создание класса TeamItem.

Сделала табуляцию во всем документе.
Изменила свойство на CreateDate.
Убрала поле со вторичным ключом.

OxanaPozdeeva committed:
ПЕ-21б Поздеева О.В.: Создание класса TeamItem.

Еще раз сменила табуляцию документа.
